nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -72,17 +74,65 @@
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Date;
 import java.util.List;
 import java.util.Locale;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 import static org.apache.nifi.processors.standard.util.FTPTransfer.createComponentProxyConfigSupplier;
 
 public class SFTPTransfer implements FileTransfer {
+    // The following DefaultConfig is used for configuration purposes. There are PropertyDescriptors associated with this class
+    // that allow an operator to choose various algorithms and ciphers to allow to be used for their SSH connections.
+    private static final DefaultConfig DEFAULT_CONFIG = new DefaultConfig();
```

Comment:
Instead making this a class variable, what do you think about using a static initializer block to create a temporary instance instead?

Reply:
No reason to hold on to a reference I suppose. Done.